aiida_vasp/calcs/vasp.py

```diff
@@ -73,48 +73,44 @@ class VaspCalculation(VaspCalcBase):
 
     """
 
-    default_parser = 'vasp.vasp'
```

Comment:
This is a not for myself: Ask how the new parser interface works and how to hook up the VASP  parser.